app/models/ext_management_system.rb

```diff
@@ -217,6 +217,10 @@ def self.create_discovered_ems(ost)
     end
   end
 
+  def self.raw_connect!(*params)
+    !!raw_connect(*params)
```

Comment:
My only issue is that `!` as a suffix usually implies an Exception or side effect could occur.  I'm thinking this should be `raw_connect?` instead, which implies it will return a boolean.